<?xml version="1.0"?>
<launch>
       <include file="$(find assignment_2_2024)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <node pkg="assignment_2_2024" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2024" type="go_to_point_service.py" name="go_to_point"  />
  
    <node pkg="assignment_2" type="action_client_node.cpp" name="action_client_node" output="screen" />
    <node pkg="assignment_2_2024" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2" type="last_target_service.cpp" name="last_target_service" output="screen" launch-prefix="gnome-terminal --command" />

</launch>

